README.md

```diff
@@ -25,85 +25,48 @@ It was written for the purpose of powering the nascent [Merveilles community for
 * **Low maintenance**: Cerca is architected to minimize maintenance and hosting costs by carefully choosing which features it supports, how they work, and which features are intentionally omitted
 * **RSS**: Receive updates when threads are created or new posts are made by subscribing to the forum RSS feed
 
-## Usage
+## Installation
 
-```
-cerca --help
-
-USAGE:
-  run the forum
-
-    cerca -authkey "CHANGEME"
-    cerca -dev
-
-COMMANDS:
-  adduser    create a new user
-  makeadmin  make an existing user an admin
-  migrate    manage database migrations
-  resetpw    reset a user's password
-
-OPTIONS:
-  -authkey string
-        session cookies authentication key
-  -config string
-        config and settings file containing cerca's customizations (default "cerca.toml")
-  -data string
-        directory where cerca will dump its database (default "./data")
-  -dev
-        trigger development mode
-```
```

Comment:
@vladh can you reinstate this part of the readme where the --help output is displayed? i.e. add it back in a way that gels with the `Installation` section you added? i think displaying help output like that is good for discoverability of what can be done before installing and useful as an occasional reminder :) also breaks up the readme visually